.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Snuba Checks
+on:
+  push:
+    branches:
+      - master
+      - releases/**
```

Comment:
What is this branch for?

Reply:
Copy/paste mistake. These are branches from `sentry`.